app/models/miq_report_result.rb

```diff
@@ -350,6 +350,14 @@ def self.with_current_user_groups
     current_user.admin_user? ? all : where(:miq_group_id => current_user.miq_group_ids)
   end
 
+  def self.with_chargeback
+    where(:db => Chargeback::SUBCLASSES)
```

Comment:
@jrafanie Is there possibily to get subclasses of Chargeback model (or rather ActAsModel ) by any method ?
I tried `Chargeback.descendants` or `Chargeback.subclases` but it is returning empty array until ChargebackVm and ChargebackContainerProject are not loaded.

```
0> Chargeback.subclasses
=> []
0> Chargeback.descendants
=> []

0> ChargebackVm
=> ChargebackVm

0> ChargebackContainerProject
=> ChargebackContainerProject

0> Chargeback.descendants
=> [ChargebackContainerProject, ChargebackVm]

0> Chargeback.subclasses
=> [ChargebackContainerProject, ChargebackVm]
```
thanks!